content/docs/dvclive/get-started.md

```diff
@@ -105,81 +99,108 @@ dvclive
 
 <admon type="info" icon="book">
 
-You can find more details about the structure of files generated by DVCLive in
+You can find more details about the structure of files generated by DVCLive on
 the [How it Works](/doc/dvclive/how-it-works) page.
 
 </admon>
 
-## DVC Integration
-
-To build on the previous section, you can optionally wrap our code with DVC to
-run [DVC experiments](/doc/user-guide/experiment-management/) and enable new
-ways to compare and visualize our experiments.
-
-You can find below an example `dvc.yaml` containing a single stage `train`, that
-runs the script from previous steps and tracks the
-[DVCLive outputs](/doc/dvclive/how-it-works) as `dvc metrics` and `dvc plots`:
-
-```yaml
-stages:
-  train:
-    cmd: python train.py
-    deps:
-      - train.py
-    metrics:
-      - dvclive/metrics.json:
-          cache: false
-    plots:
-      - dvclive/plots:
-          cache: false
-```
+### Track the results
 
-### Run DVC Experiments
+Unless you are:
 
-You can now
-[run experiments](/doc/user-guide/experiment-management/running-experiments)
-with `dvc exp run`:
+- A)
+  [Running DVC experiments](/doc/user-guide/experiment-management/running-experiments)
+  and DVCLive is used inside a stage of `dvc.yaml`.
+- B) Passing [`save_dvc_exp=True`](/doc/dvclive/api-reference/live#parameters)
+  to [`Live()`](/doc/dvclive/api-reference/live).
 
-```cli
-$ dvc exp run
-Running stage 'train':
-> python train.py
-...
-```
+Upon each execution, DVCLive will **overwrite** the results in the workspace.
+
+You would have to use one of the options above or take care of tracking the
+results with `git` between executions.
+
+## Monitor the training
```

Comment:
We could probably make a section like "Viewing the results" and make this section and the others below subsections under that one.

Reply:
This is definitely more minor and not a blocker.

Reply:
I went with individual headers because the table of contents on the right doesn't show any header lower than 2